gitops/azure-devops/build.sh

```diff
@@ -130,21 +144,26 @@ function git_commit() {
     git checkout master
     echo "GIT STATUS"
     git status
+    echo "GIT REMOVE"
+    rm -r -- ./*/
```

Comment:
@samiyaakhtar what does the `--` do?

Reply:
It's just for separation, I can get rid of it